MixpanelDemo/MixpanelDemo/AppDelegate.swift

```diff
@@ -32,15 +32,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         Mixpanel.mainInstance().identify(
             distinctId: Mixpanel.mainInstance().distinctId)
-        Mixpanel.mainInstance().people.set(properties: ["$name": "Max Panelle"])
+        Mixpanel.mainInstance().people.set(properties: ["$name": "Max Pançelle"])
         return true
     }
 
     func applicationDidBecomeActive(_ application: UIApplication) {
         Mixpanel.mainInstance().time(event: "session length")
     }
 
-    func applicationWillTerminate(_ application: UIApplication) {
+    func applicationWillResignActive(_ application: UIApplication) {
```

Comment:
just a thought: since this might be tracked automatically, we could add a comment saying that if pre-track is enabled this is not needed. Or just remove this example?